commands/version.go

```diff
@@ -41,14 +42,11 @@ func VersionCmd() *core.Command {
 }
 
 func RunVersion(c *core.CommandConfig) error {
-	err := c.Printer.Print(rootCmd.Command.Version)
-	if err != nil {
-		return err
-	}
-	c.Printer.Verbose("sdk-go " + sdkcompute.Version)
-	c.Printer.Verbose("sdk-go-dbaas-postgres " + sdkpostgres.Version)
-	c.Printer.Verbose("sdk-go-auth " + sdkauth.Version)
-	c.Printer.Verbose("sdk-go-cert-manager " + sdkcertmanager.Version)
+	fmt.Fprintf(c.Command.Command.OutOrStdout(), jsontabwriter.GenerateLogOutput(c.Command.Command.Version))
```

Comment:
This doesn't work for some reason.

```
 % i version

```

Reply:
Found it

```suggestion
	fmt.Fprintf(c.Command.Command.OutOrStdout(), jsontabwriter.GenerateLogOutput(rootCmd.Command.Version))
```

But wow, having `Version` defined on every single command is truly... something. We should get rid of that.

Reply:
🤦🏻‍♀️ 
yeah, we really should get that fixed